acceptance-tests/xfcc_test.go

```diff
@@ -137,7 +147,8 @@ var _ = Describe("forwarded_client_cert", func() {
 		closeLocalServer, localPort, err = startLocalHTTPServer(func(w http.ResponseWriter, r *http.Request) {
 			fmt.Println("Backend server handling incoming request")
 			recordedXFCCHeader = r.Header.Get("X-Forwarded-Client-Cert")
-			w.Write([]byte("OK"))
+			recordedHeaders = r.Header
+			_,_ = w.Write([]byte("OK"))
```

Comment:
Some minor whitespace issues in here and elsewhere in this file -- can be fixed with `go fmt` (VSCode's `editor.formatOnSave` will also do this automatically)

Reply:
done